src/mount_efs/__init__.py

```diff
@@ -385,13 +385,25 @@ def start_watchdog(init_system):
         sys.stderr.write('%s\n' % error_message)
         logging.warning(error_message)
 
+def create_state_file_dir(config, state_file_dir):
+    mode = 0o750
+    try:
+        mode_str = config.get(CONFIG_SECTION, "state_file_dir_mode")
+        try:
+            mode = int(mode_str, 8)
+        except ValueError:
+            logging.warn('bad state_file_dir_mode "%s" in config file', mode_str)
```

Comment:
Capitalize the "b" in "Bad", and log the config file path as well (defined at the top as `CONFIG_FILE`)